Sources/FluentKit/Schema/SchemaBuilder.swift

```diff
@@ -38,6 +38,39 @@ public final class SchemaBuilder {
         ))
         return self
     }
+
+    public func foreignKey(_ field: String, references parent: (table: String, field: String), onUpdate: DatabaseSchema.Constraint.ForeignKeyAction = .noAction, onDelete: DatabaseSchema.Constraint.ForeignKeyAction = .noAction) -> Self {
```

Comment:
What would you think about these labels:
```swift
func foreignKey(
    _ field: String, 
    references foreignSchema: String, _ foreignField: String, 
    onUpdate: DatabaseSchema.Constraint.ForeignKeyAction = .noAction, 
    onDelete: DatabaseSchema.Constraint.ForeignKeyAction = .noAction
)
```

Usage would look like:

```swift
database.schema("foos").foreignKey("bar_id", references: "bars", "id")
```

That reads a bit nicer to me at least.

Reply:
No strong preference from me so I'll gladly make the change, especially since `SQL-Kit` already spells analogous things in the way you are proposing here.

Call site used to read
```swift
database.schema("foos").foreignKey("bar_id", references: (table: "bars", field: "id"))
```
Now it will read (as you say above)
```swift
database.schema("foos").foreignKey("bar_id", references: "bars", "id")
```

In addition to following `SQL-Kit` convention more closely, this could make the appearance of a tuple-based approach to composite foreign keys look _very_ familiar to those who know the underlying SQL calls (granted we are still deferring on support for composite foreign keys at this time):
```swift
database.schema("foos").foreignKey(("bar_id", "bar_name"), references: "bars", ("id", "name"))
```

Reply:
+1 to both things here. That is a good potential candidate for composite api. marking this as resolved :)